src/redux/sagas/motions/claimMotionRewards.ts

```diff
@@ -39,83 +34,41 @@ function* claimMotionRewards({
   const apolloClient = getContext(ContextModule.ApolloClient);
 
   try {
-    const colonyManager = yield getColonyManager();
```

Comment:
The following logic checks, of the motion ids provided to the saga, which can still be claimed by the user, and on which side there's a reward. 

The difference is that I've updated it to verify this with the db, instead of the chain. Reasoning being, querying the db is faster than the chain (I'm assuming).